h/accounts/schemas.py

```diff
@@ -189,6 +189,11 @@ class EditProfileSchema(CSRFSchema):
         default='',
         missing=colander.null
     )
+    emailAgain = colander.SchemaNode(
```

Comment:
Hmm. I could have sworn there was a deform widget type that would just render two fields and automatically set up validation that they were the same...

Yes, [there is](https://deform.readthedocs.org/en/latest/api.html#deform.widget.CheckedInputWidget).

We might not need to do any of the validation ourselves if we just set the `widget` kwarg on the email node to one of these.